packages/components/src/resizable-box/stories/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { boolean, number, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import ResizableBox from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+export default { title: 'ResizableBox', component: ResizableBox };
+
+const Example = ( props ) => {
+	const [ attributes, setAttributes ] = useState( { height: 200, width: 400 } );
+	const { height, width } = attributes;
+	const { content, ...restProps } = props;
+
+	return (
+		<div style={ { margin: 30 } }>
```

Comment:
What do you think about refactoring Example to be `ResizableBoxWithState` to return only this part:
```js
<ResizableBox
	{ ...restProps }
	size={ {
		height,
		width,
	} }
	onResizeStop={ ( event, direction, elt, delta ) => {
		setAttributes( {
			height: parseInt( height + delta.height, 10 ),
			width: parseInt( width + delta.width, 10 ),
		} );
	} }
>
	{ children }
</ResizableBox>
```
and moving unrelated styling to the parent component?

This way we could see more code in the preview in Storybook.